ostree-ami-image.sh

```diff
@@ -49,12 +49,17 @@ case "${ID}-${VERSION_ID}" in
     "centos-9")
         OSTREE_REF="centos/9/${ARCH}/edge"
         SYSROOT_RO="true"
+        ANSIBLE_OS_NAME="rhel-edge"
         ;;
     *)
         echo "unsupported distro: ${ID}-${VERSION_ID}"
         exit 1;;
 esac
 
+if [[ "${ID}-${VERSION_ID}" == "rhel-9.5" ]]; then
```

Comment:
Better to add a rhel 9.5 switch case and then set os_name like what you did in other cases.

Reply:
Done, thanks!